### ISSUE TITLE
Tidy up PG Methods

### ISSUE THREAD

**Issue by dkasabovn** [dkasabovn/gamma#8]
Within our pg methods some of the sql statements are getting messy.

I want you to tidy them up reordering the models and sql statements. 

For example:
```go
type User struct {
	Id           int
	Uuid         string
	Email        string
	PhoneNumber  string
	ImageUrl     string
	FirstName    string
	LastName     string
	UserName     string
	PasswordHash string
}
```
Is out of wack in terms of ordering. We should put all fields within user in order of importance. I would reorder from top to bottom in this way: `Id, Uuid, PasswordHash, Email, PhoneNumber, UserName, FirstName, LastName, ImageUrl`

In essence we want to make sure that the top of the struct contains the most important fields and the bottom contains less important fields.

From here we need the order of these fields to correspond with our sql statements.

Here is the current code for getting a user:
```go
func (u *userRepo) GetUser(ctx context.Context, uuid string) (*bo.User, error) {
	statement := "SELECT uuid, email, first_name, last_name, username, phone_number, image_url FROM users WHERE uuid = $1"
	res := u.dbInstance.QueryRowContext(ctx, statement, uuid)

	if res.Err() != nil {
		log.Errorf("could not get user by uuid: %s", res.Err().Error())
		return nil, res.Err()
	}

	var user bo.User

	if err := res.Scan(
		&user.Uuid,
		&user.Email,
		&user.FirstName,
		&user.LastName,
		&user.UserName,
		&user.PhoneNumber,
		&user.ImageUrl,
	); err != nil {
		log.Errorf("could not scan res into user object: %v", err)
		return nil, err
	}

	return &user, nil
}
```
The sql statement does not list out columns in the same order as our struct. I.E. in our current struct `first_name` does not come after `email`.

This is quite messy. The same goes with the order of the scan parameters.

Basically what I want you to do is to go through objects in app/domain/bo and reorder their values, then fix our sql statements and scans to reflect this new order.

You can start with user, but in the end you should refactor everything.
